registry/super_editor.test

```diff
@@ -0,0 +1,38 @@
+# Tests for the super_editor mono repo.
+
+# Contacts
+contact=flutterbountyhunters@gmail.com
+
+# Update
+update=attributed_text/
+update=super_text_layout/
+update=super_editor/
+
+# Fetch the super_editor mono repo.
+fetch=git clone https://github.com/superlistapp/super_editor.git tests
+fetch=git -C tests checkout 004a6b93a56f18ec5320d340f61f8158febd59f8
+
+# Run the tests.
+test.posix=flutter_test_repo_attributed_text_test.sh
+test.windows=flutter_test_repo_attributed_text_test.bat
+
+test.posix=flutter_test_repo_super_text_layout_test.sh
+test.windows=flutter_test_repo_super_text_layout_test.bat
+
+test.posix=flutter_test_repo_super_editor_test.sh
+test.windows=flutter_test_repo_super_editor_test.bat
+
+# To test your tests, check out the flutter/flutter repository and
+# then, from the root of that repository, run:
+#
+#   bin/flutter; cd dev/customer_testing; flutter pub get; time ../../bin/cache/dart-sdk/bin/dart run_tests.dart --repeat=100 <path>
+#
+# ...where <path> is the path to this file.
+#
+# This should run with no output, without failing, and should in total
+# take less than 15 minutes (a hundred runs of your tests taking just
```

Comment:
Did you try running them locally as per this comment? I tried running the tests locally but they seem to take a long time to run (they haven't finished yet, and I'm pretty sure we're beyond 15 minutes).

Reply:
When I ran locally they seemed to hang forever. But even in CI they do take a long time. I'm not sure why. Do you have any recommendations for figuring out what's taking so long and how to reduce the time? I don't think we're running any golden tests, at least that I recall. And the vast majority of tests are either Dart tests or pretty short widget tests...

Reply:
Ok, they took 30 real-time minutes to run 100 times on my machine (163 CPU minutes).

Let me take a quick look at the tests to see if there's anything obvious there. In general I would expect slow tests to mean either that there's a lot of tests, that the tests do a lot of looping, or that the tests wait real time rather than relying on the async systems.

Reply:
It's just that you have a lot of tests. I scanned through them and they seem very valuable and worth having despite the cost, so don't worry about it. I didn't see any red flags.

Reply:
Great. Thanks. I'm going to try to relocate and possibly de-dup these scripts before merging. So please expect that after approval, or you can hold approval until then, if you want to.